lib/plain/endpoints/common-types.ts

```diff
@@ -1,5 +1,9 @@
 import { QueryOptions, PaginationQueryOptions } from '../../common-types'
 
+export type GetSpaceParams = { spaceId: string }
+export type GetEnvironmentParams = { environmentId: string }
```

Comment:
I ALWAYS see it used in conjunction with `GetSpaceParams` so I'd suggest to fall back to what was there earlier: `GetEnvironmentParams = GetSpaceParams & ...`. Environment never ever works without a space so I'd bake that into the types.

Reply:
I actually separated them cause the name of this type was misleading and confusing to me. It's either should be `GetSpaceEnvironmentParams` or have only `{ environmentId: string }` as I implemented it here. What do you think @anho ?

Reply:
That would make it more explicit which I would be fine with. But for me environments don't exist without a space semantically so I'm not confused 😄